### ISSUE TITLE
Uncaught errors",{"name":"Error","message":"Object has been destroyed

### ISSUE THREAD

**Issue by jcald1** [a85/POSTMan-Chrome-Extension#7593]
Tested on both 7.10.0 and  PostmanCanary-linux-x64-7.11.0-canary01. I get the same error but only see the error message in the Canary version main.log.   I'm using the native app on Linux.  

I ran the collection runner and selected "Keep environment variables".  The app crashes shortly thereafter.  The API is returning JSON responses that are about 5.5MB with a few very large strings.

I've attached the log file, global variables, and collection file.

[main.log](https://github.com/postmanlabs/postman-app-support/files/3816760/main.log)
[files.zip](https://github.com/postmanlabs/postman-app-support/files/3817114/files.zip)

**Comment by kamalaknn** [comment]
This looks like a duplicate of #5848

**Comment by jcald1** [comment]
@kamalaknn , I don't think so.  I didn't click on Cancel and the app still crashes.  Also, according to the #[5799](https://github.com/postmanlabs/postman-app-support/issues/5799), which is #5848 is a duplicate of, that issue was fixed in the Canary version.  I'm using that version and I get this error.

**Comment by saswatds** [comment]
Hi @jcald1 the issue #5848 was incorrectly triaged as a duplicate of #5799. The later has been fixed but there is still another issue as stated in #5848 that causes the runner to crash. 

1. We have identified that this particular error appears whenever the runner window is closed (manually or otherwise) in the middle of the collection run.

2. There is also a hard limit on the amount of data that can be stored in environments that are around ~10MB. 

3. We have also seen that when really large collection runs are stored/presisted after run it crashes the app.

All of these issues are being tracked and work is ongoing to fix them on priority. If you are facing something other than these problems, feel free to report them with steps to reproduce, we will take a look.
